build/package.proj

```diff
@@ -44,6 +47,38 @@
 
     <MSBuild Projects="@(WebSdkPackageProject)"
          Targets ="Restore;Build;Pack"/>
-    
+
+  </Target>
+
+  <Target Name="_GenerateBundledVersionsProps">
+    <PropertyGroup>
+      <BundledVersionsPropsFileName>BundledVersions.props</BundledVersionsPropsFileName>
+      <BundledVersionsPropsDirectory>$(WebSdkRoot)/src/Web/Microsoft.NET.Sdk.Web.Targets/</BundledVersionsPropsDirectory>
```

Comment:
This is the simplest way to generate and pack this file though I don't like the fact that this will generate a file in src/ which is tracked by version control by default. I'm not sure I want to setup the infrastructure to pack in a different working directory to add more complexity to this PR but maybe I should add the specific file to .gitignore?